src/sentry/migrations/0873_convert_org_saved_searches_to_views.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 5.1.7 on 2025-04-28 16:43
+
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.models.savedsearch import Visibility
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def convert_org_saved_searches_to_views(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    SavedSearch = apps.get_model("sentry", "SavedSearch")
+    GroupSearchView = apps.get_model("sentry", "GroupSearchView")
+
+    org_saved_searches = SavedSearch.objects.filter(visibility=Visibility.ORGANIZATION)
+
+    for saved_search in RangeQuerySetWrapperWithProgressBar(org_saved_searches):
+        GroupSearchView.objects.update_or_create(
```

Comment:
`get_or_create` might be better here, so that if you have to re-run it you don't run a bunch of empty updates. Nbd either way though